scripts/NextJs/configs/next.config.js

```diff
@@ -7,18 +7,34 @@ const { withSentryConfig } = require('@sentry/nextjs');
 
 const moduleExports = {
   // Your existing module.exports
+
+  sentry: {
+    // Use `hidden-source-map` rather than `source-map` as the Webpack `devtool`
+    // for client-side builds. (This will be the default starting in
+    // `@sentry/nextjs` version 8.0.0.) See
+    // https://webpack.js.org/configuration/devtool/ and
+    // https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#use-hidden-source-map
+    // for more information.
+    hideSourceMaps: true,
+  },
 };
 
 const sentryWebpackPluginOptions = {
-  // Additional config options for the Sentry Webpack plugin. Keep in mind that
-  // the following options are set automatically, and overriding them is not
-  // recommended:
-  //   release, url, org, project, authToken, configFile, stripPrefix,
-  //   urlPrefix, include, ignore
+  // Additional config options for the Sentry Webpack plugin. For all available
+  // options, see: https://github.com/getsentry/sentry-webpack-plugin#options.
+
+  // Keep in mind that the following options are set automatically, and
+  // overriding them is not recommended unless you know what you're doing:
+  //   authToken, configFile, dryRun, include, ignore, org, project, release,
+  //   stripPrefix, url, urlPrefix,
+
+  // (If you do want to override which files are included and ignored, don't use
+  // `ignore` or `ignoreFile` directly. Instead, pass an array of objects of the
+  // form { paths: ['path/to/include'], ignore: ['path/to/ignore'] } to the
+  // `include` option. See
+  // https://github.com/getsentry/sentry-webpack-plugin#include.)
```

Comment:
> If you do want to override which files are included and ignored

We should clarify for what exactly the files are included/ignored.

Reply:
Wait, now I'm confused. I didn't add that part in this PR, at least not intentionally... I would have said it was already there... 🤔